.bazelrc

```diff
@@ -19,3 +21,8 @@ build \
 build:ios --define=manual_stamp=manual_stamp
 
 build:android --fat_apk_cpu=x86,x86_64,armeabi-v7a,arm64-v8a
+
+build:release \
+  --config=sizeopt \
+  --copt -ggdb3 \
+  --ios_multi_cpus=i386,x86_64,armv7,arm64
```

Comment:
I _think_ this can be updated to use `---config=fat` now. If not we should add Android's archs in here also (since it's no longer fat by default).

Reply:
Yep, should be able to